lib/collection/src/shards/local_shard/mod.rs

```diff
@@ -69,6 +73,7 @@ pub struct LocalShard {
     pub(super) path: PathBuf,
     pub(super) optimizers: Arc<Vec<Arc<Optimizer>>>,
     pub(super) optimizers_log: Arc<ParkingMutex<TrackerLog>>,
+    clock_map: Arc<Mutex<ClockMap>>,
```

Comment:
I'd prefer a name there that describes what we use it for, not a name that matches the type definition.

```suggestion
    /// Clock map holding the last seen clock tick for all clocks we know about
    last_seen_clocks: Arc<Mutex<ClockMap>>,
```

or something different.

Reply:
Sure, works for me.